encryption/encryption.cc

```diff
@@ -15,6 +15,17 @@ namespace rocksdb {
 namespace encryption {
 
 namespace {
+// For some files, we don't require them to be encrypted.
+bool skip_encryption(const std::string& fname) {
+  std::string skip_name = "CURRENT";
```

Comment:
1. Define a constant for the current file in filename/filename.h. Also update that file to use the constant.
2. Add comment why we want to exclude the current file from encryption.

Reply:
also kEncryptionSkipSuffix is a better name for it.